### ISSUE TITLE
Power management: Suspend after x minutes of inactivity

### ISSUE THREAD

**Issue by rdorsch** [LubosD/twinkle#123]
Hi,

KDE (and other desktops) support in their power management configuration to trigger some actions after a configurable time of inactivity (e.g. suspend, hibernate, or shutdown). This is certainly not wanted during a phone call, in particular since during talking inactivity at the keyboard or mouse is not uncommon.

If I remember right, vlc blocks during playing videos, amarok during audio playback, twinkle should do the same during phone calls.

Does anybody know, how inactivity is detected and what amarok, vlc,... are doing to block this?

Thanks
Rainer

**Comment by rdorsch** [comment]
A description of the mechanism used to inhibit unwanted power states during phone calls is decribed here

https://www.freedesktop.org/wiki/Software/systemd/inhibit/

**Comment by fbriere** [comment]
I have now submitted #154, which adds an option to prevent the session from being marked as idle while a call is in progress.  (That option can be found at the bottom of the "General" settings page.)

I should point out that I have only tested this with Xfce (which uses the same interface as KDE).  While the GNOME/MATE equivalent should work just as fine in theory, I'd be glad to hear a confirmation of this from actual users.

(And thank you @rdorsch for taking the time to dig out that information.  While the _systemd_ interface is not the one appropriate for this task, knowing about its existence was a good starting point.)
